.github/workflows/test.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
+    branches: '*'
```

You want `**` so that it matches branches with `/` in their name
```suggestion
    branches: '**'
```